concent_api/verifier/tests/test_utils.py

```diff
@@ -1,33 +1,200 @@
         self.assertEqual(result, False)
 
     def test_that_are_image_sizes_and_color_channels_equal_should_return_false_if_color_channels_are_not_equal(self):
-        image1 = mock.create_autospec(spec=ndarray, spec_set=True)
-        image2 = mock.create_autospec(spec=ndarray, spec_set=True)
-        image1.shape = (2000, 3000, 3)
-        image2.shape = (2000, 3000)
-        result = are_image_sizes_and_color_channels_equal(image1, image2)
+        result = are_image_sizes_and_color_channels_equal(self.image, self.image_diffrent_color_channel)
         self.assertEqual(result, False)
 
     def test_that_are_image_sizes_and_color_channels_equal_should_return_true_if_sizes_in_pixels_are_equal(self):
-        image1 = mock.create_autospec(spec=ndarray, spec_set=True)
-        image2 = mock.create_autospec(spec=ndarray, spec_set=True)
-        image1.shape = (2000, 3000, 3)
-        image2.shape = (2000, 3000, 3)
-        result = are_image_sizes_and_color_channels_equal(image1, image2)
+        result = are_image_sizes_and_color_channels_equal(self.image, self.image)
         self.assertEqual(result, True)
+
+    def test_that_parse_result_files_with_frames_function_should_return_correct_dict(self):
+        parsed_files_to_compare = parse_result_files_with_frames(
+            frames=self.frames,
+            result_files_list=self.result_files_list,
+            output_format=self.output_format,
+        )
+
+        self.assertEqual(parsed_files_to_compare, self.parsed_files_to_compare)
+
+    def test_that_parse_results_files_with_frames_function_should_return_empty_dict_because_of_wrong_output_format(self):
+        parsed_files_to_compare = parse_result_files_with_frames(
+            frames=self.frames,
+            result_files_list=self.result_files_list,
+            output_format='JPG',
+        )
+
+        self.assertEqual(parsed_files_to_compare, {})
+
+    def test_that_render_images_by_frames_function_should_return_correct_output_files_names(self):
+        with mock.patch('verifier.utils.render_image', autospec=True) as mock_render_image:
+            (blender_output_file_name_list, parsed_files_to_compare) = render_images_by_frames(
+                parsed_files_to_compare=self.parsed_files_to_compare,
+                frames=self.frames,
+                output_format=self.output_format,
+                scene_file=self.scene_file,
+                subtask_id=self.subtask_id,
+                verification_deadline=None
+            )
+            self.assertEqual(self.correct_parsed_all_files, parsed_files_to_compare)
+            self.assertEqual(self.correct_blender_output_file_name_list, blender_output_file_name_list)
+            self.assertEqual(mock_render_image.call_count, 2)
+
+    def test_that_upload_blender_output_file_should_correctly_upload_files(self):
+        with mock.patch('verifier.utils.try_to_upload_blender_output_file', autospec=True) as mock_try_to_upload:
```

Comment:
You should add raising exception as `side_effect` in this mock. Currently it is not clear why exception is being raised there.